rasa/core/policies/ted_policy.py

```diff
@@ -100,7 +103,7 @@ class TEDPolicy(Policy):
         # Hidden layer sizes for layers before the dialogue and label embedding layers.
         # The number of hidden layers is equal to the length of the corresponding
         # list.
-        HIDDEN_LAYERS_SIZES: {DIALOGUE: [], LABEL: []},
+        HIDDEN_LAYERS_SIZES: {DIALOGUE: [], LABEL: [], f"{DIALOGUE}_inputs": [100]},
```

Comment:
what is `f"{DIALOGUE}_inputs"`?

Reply:
it is the dimension of the layer which embeds different types of inputs (e.g, `action_name `and `action_text`)